lib/render/RayCaster.cpp

```diff
@@ -232,27 +267,31 @@ RayCaster::UserCoordinates::GetCurrentGrid( const RayCasterParams* params,
     {
         MyBase::SetErrMsg("UserCoordinates::GetCurrentGrid() isn't on a StructuredGrid; "
                           "the behavior is undefined in this case.");
+        return 1;
```

Comment:
Per our coding conventions a function returning an integer should return a *negative* number on failure, and 0 on success. We follow the posix system call conventions in this regard